### ISSUE TITLE
v0.1.5 doesn't show anything

### ISSUE THREAD

**Issue by mdkeil** [Barma-lej/landroid-card#10]
## **Before submitting a bug**

[X] I updated to the latest version available
[X] I cleared the cache of my browser
[X] I restarted hass more than 1time after the upgrade

## **Describe the bug**

After upgrading from v0.1.4 to v0.1.5 the panel shows nothing. I tryed to deactivate "names", "status" or "toolbar" without any effect. 

[X] no errors in the debug log

solution: Downgrading back to v0.1.4

## **Versions:**

- Release: [v0.1.5]
- Browser: [HASS Android App 2022.6.0]
- HomeAssistant version: [2022.7.4] @ Docker
- HACS [1.26.2]
- Landroid Cloud: [v2.2.2]

**Comment by mitchoklemen** [comment]
reboot the HA!

**Comment by mdkeil** [comment]
> [X] I restarted hass more than 1time after the upgrade

I always reboot HA after an update.. did it more than 1time! ;)

Edit: I updated to v0.1.5 again, reboot HA and logged in via chrome browser.. same result.. the panel is completely empty.

**Comment by Barma-lej** [comment]
Can you please provide a bit more info:
Attributes of your lawnmower and error in [the console in browser debugger](https://javascript.info/debugging-chrome#console)

**Comment by S-E-B-123** [comment]
Have the same problem, since v0.1.5 the card show nothing.
Here the message from the browser console:
![image](https://user-images.githubusercontent.com/70915123/179943587-1403bb4f-676f-45b1-9cd5-3e7cb41a9883.png)

and here the Attributes:

```
battery_level: 100
battery_icon: mdi:battery
accessories: null
battery:
  cycles:
    total: 6432
    current: 0
    reset_at: null
    reset_time: null
  temperature: 40.6
  voltage: 27.17
  percent: 100
  charging: false
blades:
  total_on: 146213
  reset_at: null
  reset_time: null
  current_on: 146213
error:
  id: 0
  description: no error
firmware:
  auto_upgrade: false
  version: 5.25
locked: false
mac_address: private
model: Landroid M1200
online: true
orientation:
  pitch: 7
  roll: -1
  yaw: 4
party_mode_enabled: false
rain_sensor:
  delay: 30
rssi: 52
schedule:
  time_extension: 0
  active: true
  auto_schedule:
    settings: {}
    enabled: null
  primary:
    monday:
      start: '11:00'
      end: '19:00'
      duration: 480
      boundary: false
    tuesday:
      start: '11:00'
      end: '19:00'
      duration: 480
      boundary: false
    wednesday:
      start: '11:00'
      end: '19:00'
      duration: 480
      boundary: true
    thursday:
      start: '11:00'
      end: '18:45'
      duration: 465
      boundary: false
    friday:
      start: '11:00'
      end: '18:45'
      duration: 465
      boundary: false
    saturday:
      start: '11:00'
      end: '18:45'
      duration: 465
      boundary: false
    sunday:
      start: '11:00'
      end: '19:00'
      duration: 480
      boundary: true
serial_number: 'private'
statistics:
  worktime_blades_on: 146213
  distance: 2927109
  worktime_total: 183483
status_info:
  id: 32
  description: cutting edge
time_zone: Europe/Berlin
state_updated_at: '2022-07-20T11:00:11+02:00'
zone:
  current: 3
  next: 0
  index: 3
  indicies:
    - 0
    - 1
    - 2
    - 3
    - 0
    - 1
    - 3
    - 0
    - 1
    - 0
  starting_point:
    - 7
    - 53
    - 71
    - 109
capabilities: null
mqtt_connected: true
supported_landroid_features: 12863
device_class: landroid_cloud__state
friendly_name: Robi
supported_features: 12500
```

**Comment by mdkeil** [comment]
> Can you please provide a bit more info: Attributes of your lawnmower and error in [the console in browser debugger](https://javascript.info/debugging-chrome#console)

If you need further information, I will provide attributes and debug from browser later, when I'm home.

**Comment by Papi77** [comment]
In my case, I also see "Error: Specifying entity is required!" (although the right entity has in fact been chosen!)
![image](https://user-images.githubusercontent.com/26800136/179953087-ce573806-cb08-462d-ada2-30a8ed9e1cd9.png)

**Comment by mdkeil** [comment]
```
Uncaught (in promise) TypeError: can't convert null to object
    value landroid-card.js:1
    value landroid-card.js:1
    value landroid-card.js:1
    value landroid-card.js:1
    value landroid-card.js:1
    value landroid-card.js:1
    value landroid-card.js:1
    update landroid-card.js:1
    performUpdate landroid-card.js:1
    scheduleUpdate landroid-card.js:1
    _$Ej landroid-card.js:1
    requestUpdate landroid-card.js:1
    u landroid-card.js:1
    cp landroid-card.js:1
    Zp landroid-card.js:1
    bx landroid-card.js:1
    n landroid-card.js:1
    E typography.js:181
    t typography.js:181
    a ha-logbook.ts:42
    u ha-logbook.ts:42
    u ha-logbook.ts:42
    d ha-logbook.ts:42
    v ha-logbook.ts:42
    value hui-masonry-view.ts:44
    value hui-masonry-view.ts:44
    value hui-masonry-view.ts:44
    i typography.js:181
    u ha-logbook.ts:42
    promise callback*7778/u/< ha-logbook.ts:42
    u ha-logbook.ts:42
    d ha-logbook.ts:42
    v ha-logbook.ts:42
    value hui-masonry-view.ts:44
    _cards hui-masonry-view.ts:44
    value hui-masonry-view.ts:44
    value hui-masonry-view.ts:44
    value hui-masonry-view.ts:44
    performUpdate app.054877b3.js:1250
    scheduleUpdate app.054877b3.js:1250
    _$EC app.054877b3.js:1250
    requestUpdate app.054877b3.js:1250
    o app.054877b3.js:1250
    b app.054877b3.js:1250
    r hui-masonry-view.ts:44
    E typography.js:181
    t typography.js:181
    value c183a770.js:386
    value c183a770.js:386
    setTimeout handler*i/< typography.js:181
    i typography.js:181
    value c183a770.js:386
    _$AE app.054877b3.js:1250
    performUpdate app.054877b3.js:1250
    scheduleUpdate app.054877b3.js:1250
    _$EC app.054877b3.js:1250
    requestUpdate app.054877b3.js:1250
    o app.054877b3.js:1250
    b app.054877b3.js:1250
    w app.054877b3.js:1250
    C c183a770.js:84
    E typography.js:181
    t typography.js:181
    p app.054877b3.js:1250
    T app.054877b3.js:1250
    _$AI app.054877b3.js:1250
    S app.054877b3.js:1250
    update app.054877b3.js:1250
    performUpdate app.054877b3.js:1250
    scheduleUpdate app.054877b3.js:1250
    _$EC app.054877b3.js:1250
    requestUpdate app.054877b3.js:1250
    set app.054877b3.js:1250
    value c183a770.js:84
    value c183a770.js:84
    _$AE app.054877b3.js:1250
landroid-card.js:1:372557
```

```
battery_level: 58
battery_icon: mdi:battery-60
accessories: 
ultrasonic: true

battery: 
cycles:
  total: 219
  current: 0
  reset_at: null
  reset_time: null
temperature: 34.8
voltage: 18.19
percent: 58
charging: false

blades: 
total_on: 37242
reset_at: 34676
reset_time: '2022-07-12T19:26:44+02:00'
current_on: 2566

error: 
id: 0
description: no error

firmware: 
auto_upgrade: false
version: 3.26

locked: false
mac_address: ##########
model: Landroid L1000
online: true
orientation: 
pitch: -2
roll: -0.4
yaw: 92.9

rain_sensor: 
delay: 120
triggered: false
remaining: 0

schedule: 
time_extension: 0
active: true
auto_schedule:
  settings: {}
  enabled: null
primary:
  monday:
    start: '11:00'
    end: '13:30'
    duration: 150
    boundary: false
  tuesday:
    start: '11:00'
    end: '13:30'
    duration: 150
    boundary: false
  wednesday:
    start: '11:00'
    end: '13:30'
    duration: 150
    boundary: true
  thursday:
    start: '11:00'
    end: '13:30'
    duration: 150
    boundary: false
  friday:
    start: '11:00'
    end: '13:30'
    duration: 150
    boundary: false
  saturday:
    start: '11:00'
    end: '13:30'
    duration: 150
    boundary: false
  sunday:
    start: '11:00'
    end: '13:30'
    duration: 150
    boundary: true
secondary:
  monday:
    start: '18:00'
    end: '20:30'
    duration: 150
    boundary: false
  tuesday:
    start: '18:00'
    end: '20:30'
    duration: 150
    boundary: false
  wednesday:
    start: '18:00'
    end: '20:30'
    duration: 150
    boundary: false
  thursday:
    start: '18:00'
    end: '20:30'
    duration: 150
    boundary: false
  friday:
    start: '18:00'
    end: '20:30'
    duration: 150
    boundary: false
  saturday:
    start: '18:00'
    end: '20:30'
    duration: 150
    boundary: false
  sunday:
    start: '18:00'
    end: '20:30'
    duration: 150
    boundary: false

serial_number: ##########
status_info: 
id: 7
description: mowing

time_zone: Europe/Berlin
zone: 
current: 0
next: 0
index: 0
indicies:
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
starting_point:
  - 0
  - 0
  - 0
  - 0

capabilities: Edge Cut, One-Time-Schedule, Party Mode, Motor Torque
mqtt_connected: true
supported_landroid_features: 15359
party_mode_enabled: false
rssi: -90
statistics: 
worktime_blades_on: 37242
distance: 659076
worktime_total: 38967

torque: -10
state_updated_at: 2022-07-20T12:51:55+02:00
device_class: landroid_cloud__state
icon: mdi:robot-mower
friendly_name: Graspirat
supported_features: 12500
```

**Comment by Papi77** [comment]
Note - the mobile app (iPhone) seems to show the card correctly!

**Comment by S-E-B-123** [comment]
Not with my two IPhones :(

**Comment by mdkeil** [comment]
> Note - the mobile app (iPhone) seems to show the card correctly!

Not with my PocoF1/ Android 11.

**Comment by Papi77** [comment]
0.1.6 - still blank for me in Firefox (but still works on iPhone)

**Comment by S-E-B-123** [comment]
Nothing changed from my side, the card is always blank (PC -  FF,Edge,Vivaldi and Iphone - HA app).
browser cache cleaned after reinstall.

**Comment by mdkeil** [comment]
same error with 0.1.6 and always blank.

**Comment by Barma-lej** [comment]
Hi guys. Can you please give me a feedback, is this problem with the versions 0.1.7-0.1.8 solved?

**Comment by Papi77** [comment]
Yes, from my end it’s solved! Thanks for your help!

**Comment by Barma-lej** [comment]
Well, i think that problem was in Home Assistent user language settings. I had added default value for user language

**Comment by wimpie007** [comment]
NOT fixed here.
Card not showing up. (reboot, clear cache,...)
Unfortunately, reverting through HACS doesn't work also. (can only revert to 1.5)

**Comment by Barma-lej** [comment]
> can only revert to 1.5

The issue appeared in 0.1.5...

Pleas publish attributes of your lawnmower and error in [the console of your browser debugger](https://javascript.info/debugging-chrome#console)

**Comment by wimpie007** [comment]
![image](https://user-images.githubusercontent.com/33153228/180388215-9e1fd1f1-d0e7-473b-9a25-7218214654a1.png)

One remark: my landroid is offline for the moment.

**Comment by Barma-lej** [comment]
Fixed in 0.2.0
